README.md

```diff
@@ -1,13 +1,13 @@
-# ImpersonateCustomer
-ImpersonateCustomer is a canonical component that any VTEX app can import.
+# Telemarketing
+Telemarketing is a canonical component that any VTEX app can import.
 
 To import it into your code: 
 ```js
-import ImpersonateCustomer from 'vtex.impersonate-customer/ImpersonateCustomer'
+import Telemarketng from 'vtex.telemarketing/index'
 ```
 
```

Comment:
Add the admin setup to ability the menu.

Suggestion: The admin user has a role of `2 - Televendas (Call center operator)` in your profile access ...

Reply:
👍